wandb/sdk/wandb_run.py

```diff
@@ -1725,16 +1727,26 @@ def _on_finish(self) -> None:
             self._exit_code, settings=self._settings, printer=self._printer
         )
 
+        tic = time.monotonic()
         while not (self._poll_exit_response and self._poll_exit_response.done):
             if self._backend and self._backend.interface:
                 self._poll_exit_response = (
                     self._backend.interface.communicate_poll_exit()
                 )
-                logger.info(f"got exit ret: {self._poll_exit_response}")
+                logger.info(f"Got exit ret: {self._poll_exit_response}")
                 self._footer_file_pusher_status_info(
                     self._poll_exit_response, printer=self._printer,
                 )
             time.sleep(0.1)
+        elapsed_time = time.monotonic() - tic
+        megabyte = wandb.util.POW_2_BYTES[2][1]
+        uploaded_size_megabytes = (
+            self._poll_exit_response.pusher_stats.total_bytes / megabyte
+        )
+        wandb.termlog(
+            f"Uploaded {uploaded_size_megabytes:.5f} MB "
+            f"in {elapsed_time:.2f} seconds ({uploaded_size_megabytes / elapsed_time:.2f} MB/s)"
+        )
```

Comment:
This is the relevant part. Done in a pretty dumb way; open to suggestions on how to do it more cleanly.

Reply:
This seems fine for now, nice improvement.